File: Cliente_proyecto_final/main/java/com/example/practica3/Datos/Producto.java
package com.example.practica3.Datos;

import java.io.Serializable;

public class Producto implements Serializable {
    public int id_producto;
    public String nombre;
    public double precio ;
    public String descripcion;
    public int cantidadProducto;


    public int getId_producto() {
        return id_producto;
    }

    public String getNombre() {
        return nombre;
    }

    public double getPrecio() {
        return precio;
    }

    public String getDescripcion() {
        return descripcion;
    }

    public int getCantidadProducto() {
        return cantidadProducto;
    }

    public void setCantidadProducto(int cantidadProducto) {
        this.cantidadProducto = cantidadProducto;
    }

    public void setId_producto(int id_producto) {
        this.id_producto = id_producto;
    }
}

File: Servidor_proyecto_final/DataObjects/Producto.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;

namespace servidorProyectoComercioElecronico.DataObjects
{
    public class Producto
    {
        public int id_producto { get; set; }
        public string nombre { get; set; }
        public double precio { get; set; }

        public string descripcion { get; set; }

        public int cantidadProducto { get; set; }
    }
}